src/App.tsx

```diff
@@ -1,26 +1,162 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import * as todoService from './api/todos';
+import { Header } from './components/Header';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterTodo, setFilterTodo] = useState('All');
```

Comment:
And use enum here 
<img width="410" alt="image" src="https://github.com/user-attachments/assets/4785a80e-584f-4dbe-b46b-a0f2f7a9773c" />